app/src/main/assets/pages/Insuree.js

```diff
@@ -193,14 +193,9 @@ $(document).ready(function () {
     }
 
     $("#imgInsuree").click(function () {
-        var imagePath = Android.selectPicture();
-        if (imagePath != "") {
-            $("#hfNewPhotoPath").val(imagePath);
-            $(this).attr('src', imagePath);
-        }
+        Android.selectPicture();
```

Comment:
I assume android ask to take the photo (if not ignore this comment) ? if yes a small comment might help (it will be hard to understand later without it)
// use android to take a photo or select a picture

Reply:
Android is an alias to the ClientAndroidInterface object added as a bridge between java and JS. This call is just ClientAndroidInterface.selectPicture() method.